homeassistant/components/speedtestdotnet/config_flow.py

```diff
@@ -52,7 +53,7 @@ async def async_step_user(
 class SpeedTestOptionsFlowHandler(OptionsFlow):
     """Handle SpeedTest options."""
 
-    def __init__(self, config_entry: ConfigEntry) -> None:
+    def __init__(self, config_entry: SpeedTestConfigEntry) -> None:
```

Comment:
With that the `config_entry` type in `async_get_options_flow` should also be changed.
https://github.com/home-assistant/core/blob/64b588165268b9fb96112540f9dd09d3ffcfc3dd/homeassistant/components/speedtestdotnet/config_flow.py#L33-L35